pages/mesosphere/dcos/services/kafka/2.8.1-2.3.1/release-notes/index.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Release notes for 2.8.1-2.3.1
+title: Release Notes
+menuWeight: 5
+model: /mesosphere/dcos/services/kafka/data.yml
+render: mustache
+---
+
+## Version 2.8.1-2.3.1
+
+### Updates
+- Upgrade the base tech version of Apache Kafka to `2.3.1`. See Kafka's Release Notes for [2.3.1](https://www.apache.org/dist/kafka/2.3.1/RELEASE_NOTES.html) for details.
+
+### New Features
+- Improve status check to verify topic successfully created in Zookeeper. In case of failure, the service returns `500 INTERNAL_SERVER_ERROR` instead of `200 OK`. 
```

Comment:
Any time you have a new feature you typically want to bump the minor version too so it could be `2.9.0-2.3.1` but this 'feature' is more of a bug fix which qualifies as a dot release. I would say change [this line](https://github.com/mesosphere/dcos-docs-site/pull/2632/files#diff-5fc8072e50b238d67f51caecdfa5baf6R16) to `### Improvements`

Reply:
> Any time you have a new feature you typically want to bump the minor version too so it could be `2.9.0-2.3.1` but this 'feature' is more of a bug fix which qualifies as a dot release. I would say change [this line](https://github.com/mesosphere/dcos-docs-site/pull/2632/files#diff-5fc8072e50b238d67f51caecdfa5baf6R16) to `### Improvements`

Thanks for the suggestion :smiley: . Will update it.